dashboard/config/routes.rb

```diff
@@ -194,6 +194,9 @@ def redirect_to_teacher_dashboard
 
   resources :zendesk_session, only: [:index]
 
+  post '/report_abuse', :to => 'abuse_report#report_abuse'
+  get '/report_abuse', :to => 'abuse_report#report_abuse_form'
```

Comment:
Is there a better route for this? Not sure how we feel about adding this as a new top level route.